### ISSUE TITLE
Dont append suggestions

### ISSUE THREAD

**Issue by jchannon** [reflectiondm/vscode-classyNaming#2]
In a method if I type `IFoo f` intellisense kicks in and I see `fFoo` I think the `f` comes from what I typed an `Foo` comes from this extension.

Would it be possible to show the intellisense dropdown with the suggestions from this extension rather than append the suggestions from what the user typed in. That way I can use the arrow key and choose a suggestion

**Comment by reflectiondm** [comment]
Indeed. I should update suggestion logic to address this issue. Thanks for the report!
I'll publish an update soon.

**Comment by jchannon** [comment]
Awesome

**Comment by reflectiondm** [comment]
@jchannon The issue is fixed, please checked the updated version.
Sorry for taking so long, I had some weird troubles publishing my extension.
